processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorProperties.java

```diff
@@ -218,4 +219,10 @@
     public ProcessorProperties(Map<PropertyDefinition<?>, Property<?>> properties) {
         super(properties);
     }
+
+    public static List<Property<?>> defaultProperties() {
```

Comment:
Please give it even a short javadoc since its also a public interface.

Reply:
I added a short description for this method: 2e70198